src/file-selector.ts

```diff
@@ -121,6 +121,10 @@ function flatten<T>(items: any[]): T[] {
 }
 
 function fromDataTransferItem(item: DataTransferItem) {
+    if ('getAsFileSystemHandle' in DataTransferItem.prototype) {
+        return item.getAsFileSystemHandle().then((fileSystemHandle: any) => fileSystemHandle.getFile())
```

Comment:
I think that you could do:
```suggestion
        return (item as any).getAsFileSystemHandle().then((fileSystemHandle: any) => fileSystemHandle.getFile())
```
to get rid of the typescript error.